src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -581,7 +584,7 @@ private JabRefPreferences() {
         defaults.put(AUTOCOMPLETER_FIRSTNAME_MODE, AutoCompleteFirstNameMode.BOTH.name());
         defaults.put(AUTOCOMPLETER_FIRST_LAST, Boolean.FALSE); // "Autocomplete names in 'Firstname Lastname' format only"
         defaults.put(AUTOCOMPLETER_LAST_FIRST, Boolean.FALSE); // "Autocomplete names in 'Lastname, Firstname' format only"
-        defaults.put(AUTOCOMPLETER_COMPLETE_FIELDS, "author;editor;title;journal;publisher;keywords");
+        defaults.put(AUTOCOMPLETER_COMPLETE_FIELDS, "author;editor;title;journal;publisher;keywords;crossref");
```

Comment:
I think, `;related;entryset` is missing here. Can you add them please? `entryset` is from biblatex.

If someone else wants to check: `FieldProperty.SINGLE_ENTRY_LINK` and `FieldProperty.MULTIPLE_ENTRY_LINK` could be possible starting points.